.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - question/research
+  - feature/enhancement
+  - "waiting for dependency"
+  - "good first issue"
+  - priority
```

Comment:
I think we should remove this from `exemptLabels`, imagine we have an issue or a PR that is a priority but it is inactive for 60 days, it means either we should fix it or it is not really a priority.

Reply:
Like Wasabi is not a user-friendly issue? It would stale from time-to-time. Let us decide if something is a priority and stalebot should not be taken into consideration regarding that at any point.

Reply:
Regarding `Wasabi is not a user-friendly issue` IMO we should somehow tackle that issue or some parts of it, but I think we should remove the priority label from it, because we are not treating it as a priority issue.

Reply:
I agree.